client/src/actions/tagsFilter.js

```diff
@@ -0,0 +1,115 @@
+import {
+    FILTER_TAGS_SEARCH_PENDING, FILTER_TAGS_SEARCH_ERROR, FILTER_TAGS_SEARCH_SUCCESS,
+    FILTER_TAGS_LOAD_ERROR, FILTER_TAGS_LOAD_SUCCESS, FILTER_TAGS_LOAD_PENDING,
+    FILTER_MATERIALS_SEARCH_PENDING, FILTER_MATERIALS_SEARCH_SUCCESS, FILTER_MATERIALS_LOAD_PENDING,
+    FILTER_MATERIALS_LOAD_SUCCESS, FILTER_MATERIALS_LOAD_ERROR, FILTER_MATERIALS_SEARCH_ERROR
+} from '../constants';
```

Comment:
Is there any nicer way to do our imports in our actions?

Reply:
we could group the exports in the constants file like:

```
const FILTER_MATERIALS_LOAD = {
    PENDING: ...,
    SUCCESS: ...,
    ERROR
}
export FILTER_MATERIALS_LOAD
``

then you just need to import FILTER_MATERIALS_LOAD and use FILTER_MATERIALS_LOAD.PENDING etc.